README.md

```diff
@@ -1,237 +1,122 @@
-functionality, to gather reports on integration failures and to inform future
-improvements to the developer experience of the External Dependency Manager
-plugin. Note that the analytics collected are limited to the scope of the EDM4U
-plugin’s usage.
+## Getting Started
 
-For details of what is logged, please refer to the usage of
-`EditorMeasurement.Report()` in the source code.
+Check out [troubleshooting](troubleshooting-faq.md) if you need help.
 
-# Requirements
+### Install via OpenUPM
 
-The *Android Resolver* and *iOS Resolver* components of the plugin only work
-with Unity version 4.6.8 or higher.
+EDM4U is available on
+[OpenUPM](https://openupm.com/packages/com.google.external-dependency-manager/):
 
-The *Version Handler* component only works with Unity 5.x or higher as it
-depends upon the `PluginImporter` UnityEditor API.
+```
+openupm add com.google.external-dependency-manager
+```
 
-The *Package Manager Resolver* component only works with
-Unity 2018.4 or above, when
-[scoped registry](https://docs.unity3d.com/Manual/upm-scoped.html)
-support was added to the Package Manager.
+### Install via Google APIs for Unity
 
-# Getting Started
+EDM4U is available both in UPM and legacy `.unitypackage` formats on
+[Google APIs for Unity](https://developers.google.com/unity/archive#external_dependency_manager_for_unity).
 
-Before you import EDM4U into your plugin project, you first
-need to consider whether you intend to *redistribute* `EDM4U`
-along with your own plugin.
+You may install the UPM version (.tgz) as a
+[local UPM package](https://docs.unity3d.com/Manual/upm-ui-local.html).
 
-## Plugin Redistribution
+You can also install EDM4U in your project as a `.unitypackage`. This is not
+recommended due to potential conflicts.
 
-If you're a plugin maintainer, redistributing `EDM4U` inside your own plugin
-will ease the integration process for your users, by resolving dependency
-conflicts between your plugin and other plugins in a user's project.
-
-If you wish to redistribute `EDM4U` inside your plugin,
-you **must** follow these steps when importing the
-`external-dependency-manager-*.unitypackage`, and when exporting your own plugin
-package:
-
-   1. Import the `external-dependency-manager-*.unitypackage` into your plugin
-      project by
-      [running Unity from the command line](https://docs.unity3d.com/Manual/CommandLineArguments.html), ensuring that
-      you add the `-gvh_disable` option.
-   1. Export your plugin by [running Unity from the command line](https://docs.unity3d.com/Manual/CommandLineArguments.html), ensuring that
-      you:
-      - Include the contents of the `Assets/PlayServicesResolver` and
-        `Assets/ExternalDependencyManager` directory.
-      - Add the `-gvh_disable` option.
+### Conflict Resolution
 
-You **must** specify the `-gvh_disable` option in order for the Version
-Handler to work correctly!
+For historical reasons, a package maintainer may choose to embed EDM4U in their
```

Comment:
I don't think we should remove the instructions around redistribution via unitypackage, since a lot of people still use this method.

Reply:
I kept that section but moved it down because I believe that we should stop endorsing this method. This method solves a legacy problem and it creates more dependency conflicts than what it solves.